static/app/components/events/eventStatisticalDetector/lineChart.tsx

```diff
@@ -39,19 +40,18 @@ function LineChart({
     const needsLabel = true;
     const intervalSeries = getIntervalLine(
       theme,
-      percentileSeries || [],
+      percentileSeries,
       0.5,
       needsLabel,
       evidenceData,
       true
     );
-    return [
-      ...percentileSeries,
-      ...intervalSeries.filter(s => !s.markArea), // get rid of the shading
-    ];
+    return [...percentileSeries, ...intervalSeries];
   }, [percentileSeries, evidenceData, theme]);
 
-  const rightSeries = useMemo(() => [throughputSeries], [throughputSeries]);
+  const rightSeries = useMemo(() => {
+    return [throughputSeries];
+  }, [throughputSeries]);
```

Comment:
Need to figure out how to draw the breakpoint here

Reply:
There's a util called `getIntervalLine` a few lines up, if you make another series and apply this `markLine` property:
```
  periodDividingLine.markLine = {
    data: [
      {
        xAxis: divider,
      },
    ],
    label: {show: false},
    lineStyle: {
      color: theme.textColor,
      type: 'solid',
      width: 2,
    },
    symbol: ['none', 'none'],
    tooltip: {
      show: false,
    },
    silent: true,
  };
```

this should get you that breakpoint iirc